README.md

```diff
@@ -1,3 +1,164 @@
 # cogitate_tool
 
-Tool to analyze contributions when working in a team.
+## Development Doc
+
+After pulling the repo, use `pipenv shell` in `cogitate_tool/` to enter the virtual
+environment. Use `exit` to exit. Under the virtual environment, use
+`pipenv install <package_name> --dev` to install new packages.
+
+### 1. File Structure
+
+```bash
+.
+├── cogitate_tool.py ------------- *main function*
+├── driller.py ------------------- *PyDriller*
+├── Pipfile ---------------------- *Package file*
+├── Pipfile.lock ----------------- *Package version lock file*
+├── pyinquirer ------------------- *CLI demo*
+│   ├── hello.py ----------------- *Demo Hello World*
+│   ├── img ---------------------- *Images for markdown*
+│   │   ├── interactive.png
+│   │   └── result.png
+│   └── intactv.py --------------- *Examples for choices question*
+└── README.md
+```
+
+### 2. CLI
+
+The [homepage](https://github.com/CITGuru/PyInquirer) for `PyInquirer`. And a
+[helpful site](https://codeburst.io/building-beautiful-command-line-interfaces-with-python-26c7e1bb54df).
+
+The available attributes can be found at their homepage.
+
+#### 2.1 Hello World
+
+Basic example:
+
+```python
+from __future__ import print_function, unicode_literals
+from PyInquirer import prompt
+"""pprint print data structures in a beautiful way"""
+from pprint import pprint
+
+"""The question for users"""
+questions = [
+    {
+        'type': 'input',
+        'name': 'first_name',
+        'message': 'What\'s your first name',
+     }
+]
+
+"""Create objects for answers"""
+answers = prompt(questions)
+"""print the answer we got, the dictionary look data can be used for future use"""
+pprint(answers)
```

Comment:
I'm not sure if it's necessary to have a "hello world" section demonstrating the code here, but that's just my personal opinion.